src/borg/archiver.py

```diff
@@ -2665,42 +2687,40 @@ def do_break_lock(self, args, repository):
             # note: using = is essential to avoid cmdline argument parsing issues.
             borg create --pattern=+pics/2018/good --pattern=-pics/2018 repo::arch pics
 
+            # backup only JPG/JPEG files (case insensitive) in all home directories:
+            borg create --pattern 're:\\.jpe?g(?i)' repo::arch /home
+
             # use a file with patterns:
-            borg create --patterns-from patterns.lst repo::arch
+            borg create --patterns-from patterns.lst repo::arch /
```

Comment:
is the / required? the recursion root can also be given in the patterns.lst.

Reply:
It's not required, but showcases a patterns file without R prefix. The other example below features roots in the file only.

Reply:
I added a comment to clarify this.